domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -54,8 +54,10 @@ const val TEST_TOPIC_ID_0 = "test_topic_id_0"
 const val TEST_TOPIC_ID_1 = "test_topic_id_1"
 const val TEST_TOPIC_ID_2 = "test_topic_id_2"
 const val FRACTIONS_TOPIC_ID = "GJ2rLXRKD5hw"
-const val SUBTOPIC_TOPIC_ID = 1
+const val SUBTOPIC_TOPIC_ID_0 = 0
+const val SUBTOPIC_TOPIC_ID_1 = 1
 const val SUBTOPIC_TOPIC_ID_2 = 2
+const val SUBTOPIC_TOPIC_ID_3 = 3
```

Comment:
Are these specific to fractions? I suggest renaming them to be clearer on which topic they correspond to since they're currently quite general in their naming.

Reply:
done.